commons/commons-model-extended-mapping/src/main/kotlin/ebi/ac/uk/extended/mapping/from/ToExtFileListMapper.kt

```diff
@@ -34,14 +35,22 @@ class ToExtFileListMapper(
         target: File,
         sources: FileSourcesList,
     ): File {
+
+        fun copy(
+            input: InputStream,
+            format: SubFormat,
+            target: OutputStream,
+        ) {
+            val sourceFiles = serializationService.deserializeFileList(input, format)
+                .onEachIndexed { idx, bioFile -> logger.info { "$accNo, Mapping file $idx, path='${bioFile.path}'" } }
```

Comment:
Nice change, I wonder if we should apply the same validation to [this endpoint](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/FileListResource.kt#L34) so we don't have two separate validation workflows